test/database/cipher.go

```diff
@@ -0,0 +1,24 @@
+package database
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+var fakeCode []byte = []byte{'F', 'A', 'K', 'E'}
```

Comment:
fyi the first []byte is superfluous, `var fakeCode = []byte{'F', 'A', 'K', 'E'}` also works